src/main/webapp/jsp/instructorCourseEnroll.jsp

```diff
@@ -15,7 +15,7 @@
   <%=Const.ParamsNames.SESSION_TOKEN%>
 </c:set>
 
-<ti:instructorPage pageTitle="TEAMMATES - Instructor" bodyTitle="Enroll Students for ${data.courseId}" cssIncludes="${cssIncludes}" jsIncludes="${jsIncludes}">
+<ti:instructorPage pageTitle="Enroll Students" bodyTitle="Enroll Students for ${data.courseId}" cssIncludes="${cssIncludes}" jsIncludes="${jsIncludes}">
```

Comment:
Just highlighting that if we drop the `pageTitle` attribute, the title would include the course ID.